DoSomethingArchive/legacy-website
# Current Run Field

#### What's this PR do?

Creates a new field on campaign nodes that allows the user to select the current run associated with the campaign. Only shows campaign runs that are tied to the campaign nid as options, otherwise it shows all of them. 
#### Where should the reviewer start?

A lot of feature export stuff, and then in `dosomething_campaign.module` I handle the form altering to filter the options.
#### How should this be manually tested?

Edit or create a new campaign, and you should see a field "Current Run" under "basic info" that allows you to select a campaign run. You should only see campaign runs that are tied that campaign nid, unless no campaign runs are currently tied to it, in which case you should see all campaign runs as options.
#### Any background context you want to provide?

See the ticket.
#### What are the relevant tickets?

addresses #5877 -> @angaither i'm not sure just adding this field really fixes the issue. I'm guessing we also need to now use the new field to determine which campaign run to show when the campaign closes. Is there an issue for that?
